neo4j/src/main/java/org/apache/zeppelin/graph/neo4j/Neo4jCypherInterpreter.java

```diff
@@ -205,7 +206,9 @@ private void addValueToLine(String key, List<String> columns, List<String> line,
       if (value instanceof Collection) {
         try {
           value = jsonMapper.writer().writeValueAsString(value);
-        } catch (Exception ignored) {}
+        } catch (Exception ignored) {
+          logger.info(ignored.getMessage());
```

Comment:
might be confusing to log an exception message (that is ignored)?

Reply:
The alternative would be to put it onto the ignore list. Currently it seems that there are no items on this list so far. Question would be where to put the ignore file? Into the `_tools` directory?

Reply:
ignore file for checkstyle?
let's change this to `logger.debug("ignored exception: " + e.getMessage());` (rename the variable)